vaadin-testbench-unit-junit5/src/test/java/com/vaadin/testbench/unit/SecurityTestConfig.java

```diff
@@ -88,5 +90,32 @@ public UserDetails loadUserByUsername(String username)
                 }
             };
         }
+
+        @Bean
+        MenuAccessControl menuAccessControl() {
+            // Workaround to support both Vaadin 24.4 that does not have
+            // SpringMenuAccessControl
```

Comment:
Maybe something is missing in this comment, did it mean to mention another vaadin version as well?

Reply:
You're right, the comment is incomplete.
`SpringMenuAccessControl` has been introduced in Vaadin 24.5, but Testbench main is currently 24.4+

Reply:
updated comment